modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -338,6 +338,33 @@ def filter(df, **kwargs):
         run_and_compare(filter, data=self.data)
 
 
+class TestMultiIndex:
+    data = {"a": np.arange(24), "b": np.arange(24)}
+
+    @pytest.mark.parametrize("names", [None, ["", ""], ["name", "name"]])
+    def test_dup_names(self, names):
+        index = pandas.MultiIndex.from_tuples(
+            [(i, j) for i in range(3) for j in range(8)], names=names
+        )
+
+        pandas_df = pandas.DataFrame(self.data, index=index) + 1
+        modin_df = pd.DataFrame(self.data, index=index) + 1
```

Comment:
why +1 is needed?

Reply:
This is to force execution by OmniSci to check it is OK with mangled column names.